src/fuse_ctrl/doc/fuse_ctrl_zeroization_guide.md

```diff
@@ -0,0 +1,52 @@
+  The `ZEROIZE` command is idempotent, i.e., it can be retried multiple times such that a zeroization of an already zeroized word has no effect.
+
+  3. When firmware determines that a partition has been sufficiently zeroized, it should reset the `fuse_ctrl` such that the zeroized data is also reflected in the buffer registers.
+
+     > Side effect: Upon initialization the `fuse_ctrl` will first read the zeroization field of a partition to determine whether it is in a zeroized state. If so, no periodic consistency and integrity checks will be executed for the partition.
+
+## Zeroization Detection
+
+There are two distinct cases when the `fuse_ctrl` performs a zeroization detection by counting the number of set bits in a word and comparing the resulting value against predefined thresholds that account for potentially stuck-at-0 bits.
+
+  1. _Initialization_: As mentioned above, the periodic consistency and integrity checks need to be disabled for a zeroized partition, furthermore when reading out the content of buffered partitions, the ECC checks need to be disabled.
+  This means that the zeroization indicator field has to be checked first before performing the remaining initialization steps.
+  A partition is said to be in a zeroized state if and only if the number of set bits in the indicator field is greater or equal the macro-specific value `ZeroizationValidBound`.
+  This bound should include a margin that accounts for defects where certain fuse bits a perpetually stuck at 0.
+  2. _Scrambled Zeroization_: As the zeroization flow requires that zeroized words are returned back to firmware for inspection, there is a possibility that scrambled data is released unintentionally either through an error case or a malicious fault attack.
+  The `fuse_ctrl` guards against this case with a two-pronged approach. First, whenever the OTP macro signals an error in response to a zeroization request, no data is released.
+  Second, for scrambled partitions, the zeroized fuse is only returned if and only if the number of set bits in the 64-bit word is greater or equal `ZeroizationValidBound`.
+  Additionally, if the number of set bits is less than `ZeroizationFatalBound`, then the block will enter a terminal error state as the number of set bits cannot be explained by stuck-at-0 bits, thus indicating an active attack.
+  On average half of the bits in a 64-bit scrambled word are set. The `ZeroizationValidBound` should be chosen such that the probability that a scrambled word is accidentally recognized as zeroized is as small as possible while still accounting for potentially stuck-at-0 bits.
+  The problematic of returning a zeroized word makes it impossible to have a mechanism that has a 0% probability of ever releasing a scrambled word to firmware, however even if this low-probability case occurs, the scrambling key remains secret making it unlikely that the word can be descrambled.
```

Comment:
The problematic -> The problem

Reply:
Also, most bits will actually be one after zeroization. Even if the scrambling key was known, it was probably impossible to retrieve even parts of the secret. Because the bits zeroized to 1 change how the scrambling key impacts the bits which are stuck at 0. And you don't know which bits were zeroized and which ones were 1 before.